src/components/interaction-designer/block-types/MobilePrimitives_MessageBlock.vue

```diff
@@ -52,13 +55,17 @@ const builderVuexNamespace = namespace('builder')
     BlockSemanticLabelEditor,
     FirstBlockEditorButton,
     BlockId,
+    TitleEditor,
```

Comment:
@poojahpatil90 We've just discussed this in the FLOIP meeting and to avoid confusion we want to ensure these editor components reflect the name in the spec. We then want to change the labels of these fields using localisation only. Can we change back Title Editor, Block Code Editor and anything else which is similar?

CC @bzabos

Reply:
Agree with Jacob here, 
Title is `label`, Code is `name` right ?

Reply:
Fixed!

@safydy-r you are right. Title is `label` and Code is `name`